components/index/CardsReference/DeathsByDeathDate/Card.vue

```diff
@@ -19,6 +19,16 @@
                 )
               }}
             </li>
+            <i18n
+              tag="li"
+              path="死亡日不明者はグラフから除いているため、累計値が「{confirmed}」の死亡者数と一致しない"
+            >
+              <template #confirmed>
+                <app-link to="/cards/details-of-confirmed-cases/">
```

Comment:
```suggestion
                <app-link :to="localePath('/cards/details-of-confirmed-cases/')">
```

リンクをlocalePathで多言語化してもらえると助かります！

Reply:
ありがとうございます。
commitしました。

Reply:
あ、すみません、prettierに引っかかったみたいです。lint通してもらえると嬉しいです。